.github/workflows/operator_ci.yaml

```diff
@@ -159,18 +158,17 @@ jobs:
       - name: Wait for k8s cluster to be ready
         run: |
           build/ci/github_actions/wait_for_k8s_ready.sh
-      - name: Deploy ibm block csi driver
+      - name: Deploy ibm block csi operator
         run: |
-          build/ci/github_actions/driver/deploy_driver.sh
+          build/ci/github_actions/driver/deploy_operator.sh
```

Comment:
Still weird to see the driver dir with an operator-related script.

Reply:
why is it under the driver dir?